modules/osd-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * networking/proxies/configuring-cluster-wide-proxy.adoc
+
+[id="osd-cluster-wide-proxy-config_{context}"]
+= Configuring a cluster-wide proxy using {product-title}
+
+You can add a proxy to your cluster when you are creating a new one. Prior to installation, however, you should verify that the proxy is accessible from their intended cluster virtual private cloud (VPC) and its private subnets.
+
+.Procedure
+To create a cluster with a proxy, run the following command:
+[source, terminal]
+----
+ocm create cluster \
+ <other arguments here> \
+ --additional-trust-bundle-file $CA_BUNDLE_FILE \
+ --http-proxy $HTTP_PROXY \
+ --https-proxy $HTTPS_PROXY
```

Comment:
This card has not been implemented yet. It isn't ok to expose this function.

Reply:
@yingzhanredhat it is the expectation that when this feature and documentation goes live, both `ocm` and `rosa` CLI will support it. We have confirmation that `ocm` is expected to match the arguments of `rosa` so I think it is safe to leave in as part of this PR.